<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="27" started-at="2016-12-22T10:59:53Z" finished-at="2016-12-22T10:59:53Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="27" started-at="2016-12-22T10:59:53Z" finished-at="2016-12-22T10:59:53Z">
      <class name="testscripts.TC001_TestLogin">
        <test-method status="FAIL" signature="setup()[pri:0, instance:testscripts.TC001_TestLogin@4631f1f8]" name="setup" is-config="true" duration-ms="0" started-at="2016-12-22T16:29:53Z" finished-at="2016-12-22T16:29:53Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at java.util.Properties$LineReader.readLine(Properties.java:434)
	at java.util.Properties.load0(Properties.java:353)
	at java.util.Properties.load(Properties.java:341)
	at testBase.TestBase.loadPropertiesFile(TestBase.java:27)
	at testBase.TestBase.init(TestBase.java:20)
	at testscripts.TC001_TestLogin.setup(TC001_TestLogin.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="testLogin()[pri:0, instance:testscripts.TC001_TestLogin@4631f1f8]" name="testLogin" duration-ms="0" started-at="2016-12-22T16:29:53Z" finished-at="2016-12-22T16:29:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
      </class> <!-- testscripts.TC001_TestLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
